src/utils/getHyperLinkFromPath.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * generates the clickable link to a specific folder
+ * This will recieve the path of a specific folder
+ * Gets the number of '/' chars in the path
+ * If there is no '/' then nothing changes since the path is same as deepestDirName
+ * If there are '/'s then it sets the name of the folder as the deepestDirName
+ * Makes a const name hyperLink put the path and deepestDirName inside the github hyperlink format
+ * returns the hyperLink
```

Comment:
this is not JSDOC formatted

Reply:
Also try to show an example with the documentation of input versus output, where is @params

Reply:
Am I doing right?